CellFabric/Cell_Fabric_FinFET__Mock/fabric_CMB5_NMOS.py

```diff
@@ -0,0 +1,226 @@
+import sys
+import json
+import argparse
+from os import system
+import gen_gds_json
+import gen_lef
+import itertools
+
+from cell_fabric import Via, Region, Canvas, Wire, Pdk
+from cell_fabric import CenterLineGrid, UncoloredCenterLineGrid, ColoredCenterLineGrid
+from cell_fabric import Grid, EnclosureGrid, SingleGrid, CenteredGrid
+
+
+class CanvasNMOS(Canvas):
```

Comment:
You are inheriting from Canvas & not DefaultCanvas. This means you are managing BEOL metal layers yourself. This is not an issue in itself but you may need to register your own postprocessors in case your VIA size is not equal to the metal overlap. To inherit from DefaultCanvas instead, look at CanvasNMOS inside tests/fabric_default.py.

Reply:
Okay, I can fix that

Reply:
Thank you ! Please review the DefaultCanvas based primitives by comparing against a golden JSON or something in case DefaultCanvas doesn't do what you are expecting it to. Marking as resolved.